docs/src/app.tsx

```diff
@@ -40,7 +41,10 @@ class App extends React.Component<any, ThemeContextData> {
           })}
         >
           <PerfDataProvider>
-            <Routes />
+            <div>
```

Comment:
If it is required two have a wrapping element:

```suggestion
            <>
```

***

_nit_